config/environments/development.rb

```diff
@@ -36,6 +36,9 @@
 
   config.action_mailer.perform_caching = false
 
+  # Mailer Devise Configuration
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
try as much as possible to not modify config/application.rb or config/initializers/* files, this is mostly due the fact that rake app:upgrade makes a diff between current version and new one when upgrading rails, everything should be done via custom initializer

Reply:
We add this to have the functionality of recover password to the admin, but i think probably we will not use it soon, so it will be commented until we need it.